styles/_typography.scss

```diff
@@ -58,8 +58,8 @@ h5 {
   }
 }
 
-p, dd {
-  font-family: var(--font-family);
+p, dd, li {
+  font-family: var(--font-family-body), sans-serif;
```

Comment:
Uses body in case this changes in future, where we have a different heading and body font